From 0a798c34ca56cc8996470d893abd0dc9d47b0458 Mon Sep 17 00:00:00 2001
From: Dang Duy Son <xpduyson@gmail.com>
Date: Sun, 27 Aug 2017 19:55:01 +0700
Subject: [PATCH 26/32] Update Security String to 2017-08-01

Applied
=================
CVE-2017-0714 A-36492637 Critical: Change 183309
CVE-2017-0722 A-37660827 Critical: Change 183309
CVE-2017-0745 A-37079296 Critical: Change 183310
CVE-2017-0726 A-36389123 High:     Change 183311
CVE-2017-0731 A-36075363 High:     Change 183312
CVE-2017-0737 A-37563942 High:     Change 183313
CVE-2017-0805 A-37237701 High:     Change 183313
CVE-2017-0738 A-37563371 High:     Change 183314/183315
CVE-2017-0713 A-32096780 High:     Change 186993/186994

Change-Id: I1f97ac78c2e2b7eef34915b13d70c2cf5044b2ba
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index b19d7ca..06c4072 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -81,7 +81,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
   # Can be an arbitrary string, but must be a single word.
   #
   # If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-  PLATFORM_SECURITY_PATCH := 2017-07-01
+  PLATFORM_SECURITY_PATCH := 2017-08-01
 endif
 
 ifeq "" "$(BUILD_ID)"
-- 
2.7.4

